scrapers/utils/bs/bs.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import logging
+
+logging.basicConfig(level=logging.ERROR)
+
+
+try:
```

Comment:
I don't think you need `try` / `except` here. Just do `import spacy`.

Reply:
i was thinking about your earlier comment for cases where you dont have spacy installed. 
sure i should remove?